.github/workflows/shared-run-e2e-tests.yaml

```diff
@@ -40,15 +40,15 @@ on:
         description: 'Config: Operator branch for config'
         type: string
         default: main
-      core_branch:
+      branch_core:
```

Comment:
Have to merge these inputs together as github has a limit of 10 inputs. See https://github.com/IABTechLab/uid2-operator/pull/889/files